lib/workers/pr/index.ts

```diff
@@ -524,3 +522,19 @@ export async function checkAutoMerge(
   logger.debug('No automerge');
   return false;
 }
+
+function sampleSize(array: string[], n: number) {
```

Comment:
maybe these types of function can be moved into somewhere in `lib/util` ? `sampleSize`, `camelCase`, etc?

Reply:
good idea